### DISCUSSION TITLE
slime-eval-last-expression-in-repl equivalent

### DISCUSSION THREAD

**Discussion by mark-gerarts** [joaotavora/sly#552] [General]
Hi, thanks for this project, it works really well!

Is there an equivalent to slime's `slime-eval-last-expression-in-repl`? Basically I'm looking for `sly-eval-defun`, but instead of printing the return value in the echo window, I'd like to have it in the repl so I can easily inspect it

**Comment by joaotavora** [comment]
Hi. I don't think there is, but you can switch to the repl quickly with C-c C-~
